### DISCUSSION TITLE
Servições Cloud

### DISCUSSION THREAD

**Discussion by leonardoelias** [frontendbr/forum#443] [Discussão]
Galera, vocês já usaram o serviço da [https://www.umbler.com/br](https://www.umbler.com/br) ?
Quais outros que vocês recomendariam ?

:)

**Comment by joao-alberto** [comment]
Eu uso, gosto muito da Umbler, tudo muito simples.

Só é uma pena que ainda não suporta Node. :(

**Comment by RatoX** [comment]
Cara eu uso o cloudatcost pra meus testes, é barata e não preciso ficar tirando do meu bolso todo mês.

**Comment by agenciawdc** [comment]
Umbler é muito bom! Para os teste gosto do Heroku, uso ele também para alguns clientes com bala na agulha _(vou até criar um tópico sobre isso)_ e para um Wordpress da vida gosto da Digital Ocean.

**Comment by joao-alberto** [comment]
Comecei a usar o [Now](https://zeit.co/now) agora e ja estou _in love_.

**Comment by mdapper** [comment]
Estou usando atualmente [DigitalOcean](https://www.digitalocean.com/). Mas requer um pouco de conhecimento de Linux, porém eles oferecem muitos tutoriais excelentes de como configurar tudo. E acho um preço muito bom. 

Usar ele para mim também foi uma vantagem que pude aprender como configurar um servidor do zero. O legal que você não fica limitado em usar o que determinada empresa de cloud te oferece, você tem um servidor Linux e instala nele o que precisar. É claro que dá um pouco de trabalho dependendo do que for, mas acho que vale o tempo investido.

**Comment by bukinoshita** [comment]
👍  no [now](https://zeit.co/now) também.

**Comment by RatoX** [comment]
@mdapper Eu fiz isso no cloudatcost, provisiono minha maquina toda com Ansible.

**Comment by brunoravanhani** [comment]
Estou só no Heroku, mas gostei do Umber, to querendo testar agora

**Comment by iamgbayer** [comment]
Não cheguei a utilizar ainda mais a [Scaleway ](http://scaleway.com) possui preços bem bacana, vale a pena dar uma conferida
